CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 
 * Fixed the unmatched language issue of some localized strings in the feedback view. ([#4007](https://github.com/mapbox/mapbox-navigation-ios/pull/4007))
 * Added `NavigationSettings.utilizeSensorData` toggle to allow navigator to use additional device sensors data for better positioning. ([#3973](https://github.com/mapbox/mapbox-navigation-ios/pull/3973))
+* Kept the road shield image resources after quiting and relaudnching the application. ([#3930](https://github.com/mapbox/mapbox-navigation-ios/pull/3930))
```

Comment:
Proposed changelog entry:
```suggestion
* Road shield images are now cached and re-used across multiple application sessions. ([#3930](https://github.com/mapbox/mapbox-navigation-ios/pull/3930))
```

Reply:
Thanks for the suggestion, the changeLog entry is added